app/assets/stylesheets/application.css

```diff
@@ -24,4 +24,5 @@
  *= require comments
  *= require wiki
  *= require question
+ *= require swagger-ui
```

Comment:
If you're directly including them down here, i don't think you need to include them via `application.css`: https://github.com/publiclab/plots2/pull/1450/files#diff-5f7846c83c0ac814e659a0a204903284R3